dashboard/app/models/pd/attendance.rb

```diff
@@ -39,6 +39,7 @@ def save_matching_enrollment_association
 
   def resolve_enrollment
     enrollment ||
+      Pd::Enrollment.with_deleted.find_by(id: pd_enrollment_id) ||
```

Comment:
This adds an extra DB query when the enrollment is soft-deleted. Is it worth removing the `enrollment` clause? That has no effect on the result, right?